crates/ty_python_semantic/src/types/function.rs

```diff
@@ -690,12 +677,10 @@ impl<'db> FunctionLiteral<'db> {
 #[derive(PartialOrd, Ord)]
 pub struct FunctionType<'db> {
     pub(crate) literal: FunctionLiteral<'db>,
```

Comment:
We might also consider consolidating `FunctionType` and `FunctionLiteral` into a single type now. (`FunctionLiteral` would have the extra fields that hold the updated signatures.)